README.md

```diff
@@ -1,313 +1,386 @@
+[![MIT License](https://img.shields.io/badge/License-MIT-green.svg)](https://choosealicense.com/licenses/mit/) &nbsp; ![GitHub last commit](https://img.shields.io/github/last-commit/paoloricciuti/sveltekit-view-transition) &nbsp; ![npm](https://img.shields.io/npm/v/sveltekit-view-transition) &nbsp; ![npm](https://img.shields.io/npm/dt/sveltekit-view-transition) &nbsp; ![npm bundle size](https://img.shields.io/bundlephobia/minzip/sveltekit-view-transition)
 
-![npm bundle size](https://img.shields.io/bundlephobia/minzip/sveltekit-view-transition)
+<br>
 
-![npm](https://img.shields.io/npm/v/sveltekit-view-transition)
-
-![npm](https://img.shields.io/npm/dt/sveltekit-view-transition)
-
-![GitHub last commit](https://img.shields.io/github/last-commit/paoloricciuti/sveltekit-view-transition)
-
-## Contributing
-
-Contributions are always welcome!
-
-For the moment there's no code of conduct neither a contributing guideline but if you found a problem or have an idea feel free to [open an issue](https://github.com/paoloricciuti/sveltekit-view-transition/issues/new)
-
-If you want the fastest way to open a PR try out Codeflow
+## Installation
 
-[![Open in Codeflow](https://developer.stackblitz.com/img/open_in_codeflow.svg)](https://pr.new/paoloricciuti/sveltekit-view-transition/)
+```bash
+npm i -D sveltekit-view-transition@latest  # or pnpm/yarn
+```
 
-## Authors
+<br>
 
-- [@paoloricciuti](https://www.github.com/paoloricciuti)
+## Usage
 
-## Installation
+### The simplest setup:
 
-Install sveltekit-view-transition with your package manager of choice
+<sub>*src/routes/+layout.svelte*</sub>
+```svelte
+<script>
+  import { setupViewTransition } from 'sveltekit-view-transition';
 
-```bash
-npm install sveltekit-view-transition@latest -D
-```
+  setupViewTransition();
+</script>
 
-```bash
-yarn add sveltekit-view-transition@latest -D
+<slot />
 ```
 
-```bash
-pnpm install sveltekit-view-transition@latest -D
-```
+This will automatically enable the default transition to every navigation.
 
-## Usage/Examples
+<br>
 
-The library propose itself to ease the common use cases for `view-transition-api`.
+### Modifying the defaults:
 
-The most easy and immediate setup is importing and adding `setupViewTransition` in the root layout page. This will automatically enable the default transition at every navigation.
+Optionally, the default animation can be modified via `::view-transition-old(root)` and `::view-transition-new(root)` like so:
 
+<sub>*src/routes/+layout.svelte*</sub>
 ```svelte
 <script>
-	import { setupViewTransition } from 'sveltekit-view-transition';
+  import { setupViewTransition } from 'sveltekit-view-transition';
 
-	setupViewTransition();
+  setupViewTransition();
 </script>
 
 <slot />
+
+<style>
+  /* Disable default crossfade. */
+  :root {
+    view-transition-name: none;
+  }
+
+  /* Or, just modify the duration. */
+  ::view-transition-old(root),
```

Comment:
This needs a `:global()` to work

Reply:
Funny, I tested this again today and realized this 😅. Good catch!